Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM    golang:1.11.0
+FROM    golang:1.11.1
```

Comment:
Just thinking of this; perhaps good to split the bump to a separate commit, so that we could build an updated / fixed 1.11.0 image as well if we see a need for it

Reply:
Sure can

Reply:
done